src/OpenTelemetry.AutoInstrumentation/Instrumentations/Logger/LoggingBuilderIntegration.cs

```diff
@@ -23,15 +23,15 @@ namespace OpenTelemetry.AutoInstrumentation.Instrumentations.Logger;
 /// Microsoft.Extensions.Logging.LoggingBuilder calltarget instrumentation
 /// </summary>
 [InstrumentMethod(
-    AssemblyName = "Microsoft.Extensions.Logging",
-    TypeName = "Microsoft.Extensions.Logging.LoggingBuilder",
-    MethodName = ".ctor",
-    ReturnTypeName = ClrNames.Void,
-    ParameterTypeNames = new[] { "Microsoft.Extensions.DependencyInjection.IServiceCollection" },
-    MinimumVersion = "3.1.0",
-    MaximumVersion = "7.*.*",
-    IntegrationName = "ILogger",
-    Type = InstrumentationType.Log)]
+    "Microsoft.Extensions.Logging",
```

Comment:
for me the previous version seems much more readable

Reply:
I would like to use required init properties. Not possible in .NET Fx.
I see benefits from having it not-nullable.

Reply:
`required init` 👍 
Just wondering maybe there is a different syntax to leave the names in front.

Reply:
We can use parameter names

like: `maximumVersion: "7.0.0"`

Reply:
aab2633b0c074971322e8cd421ae3a38141d0b27